libs/langchain/langchain/vectorstores/redis/schema.py

```diff
@@ -0,0 +1,158 @@
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from pydantic import BaseModel, Field, validator
+from redis.commands.search.field import (
+    GeoField,
+    NumericField,
+    TagField,
+    TextField,
+    VectorField,
+)
+from typing_extensions import Literal
+
+if TYPE_CHECKING:
+    from redis.commands.search.field import Field as RedisField
+
+
+class BaseField(BaseModel):
```

Comment:
most of the FieldSchemas seem identical to the redis Fields they're wrapping — what's the function of these again?